Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6@sha256:764b2c6beae96a8c8c6046392d5b1b691773337c5a7fd29347935ee0a3223146
+FROM python:3.7.6
```

Comment:
The implications of this have been discussed? I am stickler for hash chains.

Reply:
Not really. My thought process was that maybe renovate bot picks it up this way when the next point release is available, not sure tho.
Would something advice against going `python:3.7.6@sha256:xxx` or is this redundant?

Reply:
Tags can change, digests can not. If you only use the tag the image you are using can change if upstream updates the tag. In 3.6@hash, the tag should become decoration. At least that is what the documentation says.

I guess I could test that with a registry, but I trusted docker in that matter.

Reply:
I tested that.

Reply:
Alrighty did that :)